packages/typespec-vscode/test/manual/create-typespec-project-test-plan.md

```diff
@@ -0,0 +1,141 @@
+# Create TypeSpec Project from A Template Test Scenario
+
+New TypeSpec projects can be created using a variety of templates for specific purposes. TypeSpec supports the following non-branded templates:
+
+* Empty Project
+* Generic REST API
+* TypeSpec Library (with TypeScript)
+* TypeSpec Emitter (with TypeScript)
+
+Selecting a template involves:
+
+* Installing required libraries
+* Initializing essential project files with a specified folder structure
+* Configuring output settings for a designated emission purpose
+
+
+## Test Environment
+
+* OS : Windows or Linux
+* Template : Empty Project, Generic REST API, TypeSpec Library (with TypeScript), TypeSpec Emitter (with TypeScript)
+
+> Note: The extension should support all test cases in VS Code for Windows and Linux. Mac support is a stretch goal for Selenium semester.
+
+## Prerequisites
+
+Install TypeSpec Compiler before starting to write TypeSpec.
+
+* [Node.js 20+](https://nodejs.org/download/)
+* Npm 7+
+* [Install TypeSpec Compiler CLI](https://typespec.io/docs/): `"npm install -g @typespec/compiler"`
+
+## Test Steps
+
+### Step 1: Install the typespec extension.
+
+_Option 1_. Install using .vsix file: 
+   `Extension` -> `…` -> `Install form VSIX...`
+   
+   ![alt text](./images/InstallTypespec_VSIX.png)
+
+   Find the .vsix file you want to install locally.
+   
+   ![alt text](./images/InstallTypeSpec_SelectVSIXFileTest.png)
+
+_Option 2_. Install typespec with vscode extension marketplace:
+   `Extension` -> input `TypeSpec for VS Code` -> `Install`
+   
+   ![alt text](./images/InstallTypespec_ExtensionMarketplaceTest01.png)
+
+### Step 2: Trigger create TypeSpec Project
+
+_Option 1_. Clicking “Create TypeSpec Project” button/link in the “No Folder Opened” View of Explore.
+   
+   ![alt text](./images/TriggerCreateTypeSpecProject_NoFolderOpened.png)
+
+_Option 2_. Typing `> TypeSpec: Create TypeSpec Project` in the _Command Palette_.
+   
+   ![alt text](./images/TriggerCreateTypeSpecProject_CommandPalette.png)
+
+### Step 3. Select an empty folder as the root folder for the new TypeSpec project.
+   
+   ![alt text](./images/CreateTypeSpecProject_SelectFolderTest.png)
+
+### Step 4. Check if TypeSpec Compiler CLI is install. (optional)
```

Comment:
There are three path here, 
1. The compiler has been installed locally (and the local one should be used even when compiler is also installed globally)
2. The compiler has been installed globally
3. not-installed (which will trigger the compiler installation step). 
Please make sure these three path are covered.